plugin-openshift/che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/project/wizard/page/configure/ConfigureProjectViewImpl.java

```diff
@@ -220,37 +215,31 @@ public void resetControls() {
         projectsList.render(Collections.<Project>emptyList());
     }
 
-    /** {@inheritDoc} */
     @Override
     public void setExistOpenShiftProjects(List<Project> projects) {
         projectsList.render(projects);
     }
 
-    /** {@inheritDoc} */
     @Override
     public String getOpenShiftNewProjectName() {
         return osProjectNameInput.getValue();
     }
 
-    /** {@inheritDoc} */
     @Override
     public String getCheNewProjectName() {
         return cheProjectNameInput.getValue();
     }
 
-    /** {@inheritDoc} */
     @Override
     public String getOpenShiftProjectDescription() {
         return osProjectDescriptionInput.getValue();
     }
 
-    /** {@inheritDoc} */
     @Override
     public String getCheProjectDescription() {
         return cheProjectDescriptionInput.getValue();
     }
 
-    /** {@inheritDoc} */
```

Comment:
I think if we'll used some code analyzer (for example Sonar) for analyzing java doc coverage, they will say that this code doesn't have javadock. Or I'm wrong?